OloEngine/src/OloEngine/Scene/SceneSerializer.cpp

```
@@ -333,6 +335,155 @@ namespace OloEngine
             out << YAML::EndMap; // TextComponent
         }
 
+        if (entity.HasComponent<MeshComponent>())
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)